helpers/logger.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const config  = require(__dirname + '/../config/config');
+const daily   = require('winston-daily-rotate-file');
+const winston = require('winston');
+const moment  = require('moment');
+
+const stamp   = () => moment.utc().format('YYYY-MM-DD HH:mm:ss:SSS[ms]');
+
+
+// remove default
+winston.remove(winston.transports.Console);
+
+// replace with a better one
+winston.add(winston.transports.Console, {
```

Comment:
I think this comment would make more sense if combined w previous comment, and put the remove/add code together